### DISCUSSION TITLE
F1

### DISCUSSION THREAD

**Discussion by lo97** [vasqued2/ha-teamtracker#94] [Q&A]
I don't understand how to set it up in order to show F1, I can't find what name I should put in the "team" section

**Comment by vasqued2** [comment]
Use the driver’s name as the team name. For example VERSTAPPEN will track him during the race.  You can also use a * to track whoever is the race leader.

**Comment by Axydas** [comment]
Not working like that

![image](https://github.com/user-attachments/assets/a2291044-8707-49dc-bbe4-d146929df86f)

**Comment by vasqued2** [comment]
Working now.  I'm assuming you had a problem because it was too early before the race.  Let me know if you are still having problems.

![image](https://github.com/user-attachments/assets/293eacbf-4da2-4fd4-a7cb-902131b88dbc)

**Reply by Axydas** [reply]
> Working now. I'm assuming you had a problem because it was too early before the race. Let me know if you are still having problems.
> 
> ![image](https://private-user-images.githubusercontent.com/17114183/430666688-293eacbf-4da2-4fd4-a7cb-902131b88dbc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDM4OTkwMzQsIm5iZiI6MTc0Mzg5ODczNCwicGF0aCI6Ii8xNzExNDE4My80MzA2NjY2ODgtMjkzZWFjYmYtNGRhMi00ZmQ0LWE3Y2ItOTAyMTMxYjg4ZGJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDA2VDAwMTg1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQxZmUyNmM2Njg5ZjE0NmY3ZjY5MWIyOTFkY2Q5MjhmMmNiZWFhOTVhYjBjN2I4MjIyMmY3NWYwMmU2MGMzYWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rjvRZOETKR9-hL3zSwoNHghgGfo0Bb0zoXu1XQUY7YA)

Yes still

**Comment by vasqued2** [comment]
What happens if you use a wildcard for the name `*`? Show me the same screen shot at before. You are definitely calling the API as you are getting the league logo.  It's just not finding his name. If `*` works, then try just `VER`.

You will only have a little bit before it shifts to the next race and it will again not find anything until Free Practice starts for F1.
